README.md

```diff
@@ -1,105 +1,178 @@
+  implementing the CSI node services. The node component is 
+  responsible for performing the iSCSI connection management and
+  connecting to the OpenEBS Volume.
+
+OpenEBS CSI driver components can be installed by running the 
+following command. 
+
+The node components make use of the host iSCSI binaries for iSCSI 
+connection management. Depending on the OS, the spec will have to 
+be modified to load the required iSCSI files into the node pods. 
+
+Depending on the OS select the appropriate deployment file.
+
+- For Ubuntu 16.04 and CentOS.
+  ```
+  kubectl apply -f https://raw.githubusercontent.com/openebs/csi/master/deploy/csi-operator.yaml
+  ```
+
+- For Ubuntu 18.04 
+  ```
+  kubectl apply -f https://raw.githubusercontent.com/openebs/csi/master/deploy/csi-operator-ubuntu-18.04.yaml
+  ```
+
+Verify that the OpenEBS CSI Components are installed. 
 
-On deploying the app CSI Node Service receives a NodePublishVolume() request via grpc,
-which in turn patches nodeID to the previously created CVC CR and waits for the  status to be updated to bound by CVC watcher. 
-The bound status implies that the following required volume components have been created by CVC watcher:
-- Target service
-- Target deployment
-- CstorVolume CR
-- CstoVolumeReplica CR
 ```
-# Sample CVC CR after NodePublish is successful
-apiVersion: openebs.io/v1alpha1
-kind: CStorVolumeClaim
-metadata:
-  annotations:
-    openebs.io/volumeID: pvc-*
-  finalizers:
-  - cvc.openebs.io/finalizer
-  labels:
-    openebs.io/storage-pool-claim: cstor-sparse-pool
-  name: pvc-*
-  namespace: openebs
-publish:
-  nodeId: csi-node-2
-spec:
-  capacity:
-    storage:
-  cstorVolumeRef:
-  replicaCount: 
-status:
-  phase: Bound
+$ kubectl get pods -n kube-system -l role=openebs-csi
+NAME                       READY   STATUS    RESTARTS   AGE
+openebs-csi-controller-0   4/4     Running   0          6m14s
+openebs-csi-node-56t5g     2/2     Running   0          6m13s
+
 ```
-Once the status is changed to bound, steps to mount the volume are processed.
-While these steps are in progress, there might be some intermittent errors seen on describing the application pod:
+
+### Provision a cStor volume using OpenEBS CSI driver
+
+1. Make sure you already have a cStor Pool Created or you can 
+   create one using the below command. In the below spc.yaml make sure 
+   that maxPools should be greater than or equal to the number of 
+   replicas required for the volume.
+
+   The following command will create the specified number of cStor Pools
+   using the Sparse files. 
+
+   ```
+   kubectl apply -f https://raw.githubusercontent.com/openebs/csi/master/deploy/spc.yaml
+   ```
+
+2. Create a Storage Class to dynamically provision volumes 
+   using OpenEBS CSI provisioner. A sample storage class looks like:
+   ```
+   kind: StorageClass
+   apiVersion: storage.k8s.io/v1
+   metadata:
+     name: openebs-csi-cstor-sparse
+     namespace: kube-system
```

Comment:
ahh missed to add this comment earlier, SC is cluster scoped resources , so namespace not needed.